backend/tests/unit/apps/owasp/models/crp/__init__.py

```
@@ -0,0 +1 @@
+
```

Ruff (0.16.1)

[convention] Missing copyright notice at top of file

(CPY001)